functions/services/onfleet/create-task.test.js

```diff
@@ -0,0 +1,65 @@
+require("dotenv").config();
+const { createTask } = require("./index");
+
+describe("OnFleetService#createTask", () => {
+    let mockTaskCreator;
+    let fakeAddress;
+    let fakePerson;
+    let fakeNotes;
+
+    beforeEach(() => {
+        mockTaskCreator = jest.fn().mockName("mockTaskCreator");
+        fakeAddress = {
+            apartment: "",
+            state: "Missouri",
+            postalCode: "65810",
+            country: "United States",
+            city: "Springfield",
+            street: "Parkhaven Ln",
+            number: "5904",
+            unparsed: "5904 Parkhaven Ln, Springfield, MO 65810"
+        };
+        fakePerson = {
+            name: "joe ",
+            phone: "+14133333333"
+        };
+        fakeNotes = "notes";
+    });
+
+    afterEach(() => {
+        mockTaskCreator = null;
+        fakeAddress = null;
+        fakePerson = null;
+        fakeNotes = null;
+    });
+
+    it("is called with correct arguments", async () => {
+        await createTask(fakeAddress, fakePerson, fakeNotes, mockTaskCreator);
```

Comment:
You should be able to do something like the following in order to monkeypatch the create task function (instead of having to modify the params).

I tried to get below to work but was running into errors, I've never used jest so maybe you know better:
```js
const Onfleet = require("@onfleet/node-onfleet");
jest.genMockFromModule("@onfleet/node-onfleet");
const mockTasks = { create: jest.fn() };
jest.mock("@onfleet/node-onfleet", () => {
    return jest.fn().mockImplementation(() => {
        return {
            tasks: { create: jest.fn() }
        };
    });
});
```

cleanup:
```js
    beforeEach(() => {
        Onfleet.mockClear();
        mockTasks.create.mockClear();
    });
```

and then use it like normal:
```js
        await createTask(fakeAddress, fakePerson, fakeNotes);
```